src/public/cards/fileStructureExample.ts

```diff
@@ -0,0 +1,44 @@
+import {EntityType} from "@linkurious/rest-client";
```

Comment:
Are you sure we want to use typescript here?

Reply:
do you see a reason when it can be a problem ? the code is transpiled to JS when we build the project

Reply:
The code is transpiled but hard to read. If we want this to be checked by customers to understand how a plugin is made, yes, it can be a problem.

Reply:
That is an internal plugin we are not expecting from client to use it as an example to learn how to make a plugin